errors/no-server-import-in-page.md

```diff
@@ -2,13 +2,13 @@
 
 ### Why This Error Occurred
 
-`next/server` was imported in a page outside of `pages/_middleware.js` (or `pages/_middleware.tsx` if you are using TypeScript)
+`next/server` was imported in a page outside of `pages/_middleware.js` (or `pages/_middleware.ts`)
```

Comment:
A `_middleware` can appear at any level, eg. `pages/login/_middleware.{js,ts}`.

Maybe use `**` to indicate that?

Reply:
Good shout